src/main/webapp/index.html

```diff
@@ -23,7 +24,7 @@ <h1 class="logo">Poetry Critic</h1>
         </section>
         <section class="navbar-section">
             <a href="/new-poem.html" class="btn btn-link">New Poem</a>
-            <a href="#" class="btn btn-link">Search</a>
+            <a href="/about.html" class="btn btn-link">About This Project</a>
```

Comment:
I got an error clicking on the about link. shouldn't the about.html be in this PR?

Reply:
Oh this was just a slip haha. I'm currently adding the about page, and I included the link here for testing. I forgot to take it out before the PR. I will add the about page in a subsequent PR